src/Hintman/Config.hs

```diff
@@ -0,0 +1,43 @@
+module Hintman.Config
+    ( HintmanConfig(..)
+    , defaultHintmanConfig
+    , loadFileConfig
+    ) where
+
+import Toml (AnyValue (..), BiMap (..), BiToml, (.=))
+
+import qualified Toml
+
+
+
+data Check = TrailingSpaces
```

Comment:
Let's change the name of this enum to `SuggestionType`. Also, it's better to align constructors from new line, like:
```haskell
data SuggestionType
    = TrailingSpaces
    | TrailingNewlines
    deriving ...
```